src/test/mempool_tests.cpp

```diff
@@ -309,7 +309,7 @@ BOOST_AUTO_TEST_CASE(MempoolAncestorIndexingTest)
     tx2.vout[0].scriptPubKey = CScript() << OP_11 << OP_EQUAL;
     tx2.vout[0].nValue = 2 * UNIT;
     pool.addUnchecked(tx2.GetHash(), entry.Fee(20000LL).FromTx(tx2));
-    uint64_t tx2Size = GetVirtualTransactionSize(tx2);
+    uint64_t tx2Size = GetVirtualTransactionSize(tx2, entry.sigOpCost);
```

Comment:
How are these changes related?

Reply:
Because you must provide `sigOpCost` to `GetVirtualTransactionSize`. This test in master works just by lucky coincidence.